Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -2518,6 +2518,17 @@ static u32 mirror_1d(i32 coord, u32 size)
 }
 ///
 
+/// J - Restoration filters
+// J.1  General
+static ErrorOr<void> apply_restoration_filters(Frame& frame)
+{
+    auto const& frame_header = frame.frame_header;
+    if (frame_header.restoration_filter.gab || frame_header.restoration_filter.epf_iters != 0)
+        dbgln("JPEGXLLoader: FIXME: Apply restoration filters");
```

Comment:
Same question here – I get why we wouldn't want to TODO(), but not returning an error seems like it sets us up for trouble further down the road.

Reply:
For both `FIXME`s, my initial goal was to try generating the image to see if I was totally wrong or if there was some good in my other fixes. I decided to upstream it as the result was close enough to the reference image. As for the "we will forget it", I hope that the bad score on `libjxl/bench` will be enough.

I will fix the ModularLFGroups pretty soon, but I don't know when I will implement the filters.

Reply:
Should I add a way to run the decoder against `libjxl/bench` in tree?